source/frontend/src/features/mapSideBar/shared/header/HistoricalNumberSectionView.tsx

```diff
@@ -1,31 +1,77 @@
 import { useMemo } from 'react';
+import styled from 'styled-components';
 
+import ExpandableTextList from '@/components/common/ExpandableTextList';
 import { HeaderField } from '@/components/common/HeaderField/HeaderField';
+import { Dictionary } from '@/interfaces/Dictionary';
+import { ApiGen_Base_CodeType } from '@/models/api/generated/ApiGen_Base_CodeType';
 import { ApiGen_Concepts_HistoricalFileNumber } from '@/models/api/generated/ApiGen_Concepts_HistoricalFileNumber';
 import { exists } from '@/utils';
 
 export interface IHistoricalNumbersViewProps {
   historicalNumbers: ApiGen_Concepts_HistoricalFileNumber[];
 }
 
+interface HistoricalGroup {
```

Comment:
Hmm, why does this PR contain changes to the historical file numbers logic?

Reply:
Because this PR is a branch from an earlier PR (lease header updates). Once merged, the change set here will be quite small